### PR TITLE
Switched to use sciml_train

### PR DESCRIPTION
**Pull request by collinwarner**

### CODE REVIEW

File: Climate/NeuralPDE/2 [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    pre = predict_adjoint(fullp)
+    sum(abs2, training_data - pre)
+end
+
+function random_data()
+    print("here")
+    ones(N, N)
+end
+
+function cb(opt_state:: Optim.OptimizationState)
```
Comment (ChrisRackauckas) [inline Climate/NeuralPDE/2]: this isn't the right callback for sciml_train

Comment (collinwarner) [reply Climate/NeuralPDE/2]: only look at npde.jl
That is the code I am trying to get working

File: Climate/NeuralPDE/npde.jl [hunk 2]
```diff
@@ -88,45 +88,41 @@ function loss_adjoint(fullp)
...
-pl = scatter(1:n,training_data[:,10],label="data", legend =:bottomright)
-scatter!(pl,1:n,cur_pred[:,10],label="prediction",title="Spatial Plot at t=$(saveat[10])")
-pl2 = scatter(saveat,training_data[N÷2,:],label="data", legend =:bottomright, title="Time Series Plot: Middle X")
-scatter!(pl2,saveat,cur_pred[N÷2,:],label="prediction")
-plot(pl, pl2, size=(600, 300))
-savefig("npde_fit.pdf")
+#function loss_adjoint_gradient!(G, fullp)
+#    G .= Zygote.gradient(loss_adjoint, fullp)[1]
+#end
+res = DiffEqFlux.sciml_train(loss_adjoint, p, BFGS(initial_stepnorm=0.01), epochs;cb = cb,maxiters = 1000)
```
Comment (collinwarner) [inline Climate/NeuralPDE/npde.jl]: Is this right? I can't seem to get it working with the loss or predict adjoint functions

**Comment by collinwarner** [comment]
creating an updated version

**Pull request closed without merging**